From ba5d5a6e5c37c2f7c89d8af5e04dd78a68d84cae Mon Sep 17 00:00:00 2001
From: Anton Hansson <hansson@google.com>
Date: Wed, 21 Feb 2018 14:11:44 +0000
Subject: [PATCH 11/12] AdvancedDisplay: Set LOCAL_PRIVATE_PLATFORM_APIS

This change sets LOCAL_SDK_VERSION for all packages where
this is possible without breaking the build, and
LOCAL_PRIVATE_PLATFORM_APIS := true otherwise.

Setting one of these two will be made required soon, and this
is a change in preparation for that. Not setting LOCAL_SDK_VERSION
makes the app implicitly depend on the bootclasspath, which is
often not required. This change effectively makes depending on
private apis opt-in rather than opt-out.

Test: make relevant packages
Bug: 73535841
Change-Id: Id4757bcca09eda716e0a1098d9290928fa87f22d
---
 AdvancedDisplay/Android.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/AdvancedDisplay/Android.mk b/AdvancedDisplay/Android.mk
index e0d82fa..aa41fb8 100644
--- a/AdvancedDisplay/Android.mk
+++ b/AdvancedDisplay/Android.mk
@@ -6,6 +6,7 @@ LOCAL_MODULE_TAGS := optional
 LOCAL_SRC_FILES := $(call all-java-files-under, src)
 
 LOCAL_PACKAGE_NAME := AdvancedDisplay
+LOCAL_PRIVATE_PLATFORM_APIS := true
 LOCAL_CERTIFICATE := platform
 
 LOCAL_STATIC_JAVA_LIBRARIES := \
-- 
2.17.1

